frontend/src/components/Skill/SkillGroup/SkillsGroup.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, Suspense } from 'react';
+
+import { LENGTH } from 'constants/sizes';
+import Loader from 'components/Loader';
+
+import { SkillsTitle, SkillsBox, SkillsDivider } from './styled';
+import SkillItem from '../SkillItem/SkillItem';
```

Comment:
let's rebuild folder structure
folder name: SkillGroup, 
SkillGroup.tsx and its index.ts file inside and on the same level its child component's folder SkillItem
![image](https://user-images.githubusercontent.com/44285339/164185952-5c8e91f9-e5d6-4e93-8fb6-4ebea43f8dfe.png)